user_management/models/mixins.py

```diff
@@ -1,13 +1,15 @@
 from django.contrib.auth.models import BaseUserManager
-from django.contrib.auth.tokens import default_token_generator
 from django.contrib.sites.models import Site
 from django.core import checks
 from django.db import models
 from django.utils import timezone
-from django.utils.encoding import force_bytes, python_2_unicode_compatible
-from django.utils.http import urlsafe_base64_encode
+from django.utils.encoding import python_2_unicode_compatible
 from django.utils.translation import ugettext_lazy as _
-from incuna_mail import send
+
+from user_management.utils.notifications import (
+    PasswordResetNotification,
+    ValidationNotification,
+)
```

Comment:
I would do `from user_management.utils import notifications` and then `notifications.PasswordResetNotification` later.